pyorientAssignment.py

```diff
@@ -36,14 +36,26 @@
 # then test each price to see if it is smaller than the current minimum. If it is, update 
 # the minimum variable with that price. You can do something similar to find the maximum.
 
+total_every_price = 0
+min_price = 470
```

Comment:
You should set this to a very large number, incase the smallest number in the data is larger than 470.